src/url-setters.cpp

```diff
@@ -53,16 +56,17 @@ namespace ada {
 
     // Set this’s query object’s list to the result of parsing input.
     // @todo Implement this if/when we have URLSearchParams.
+    return ;
   }
 
-  void url::set_pathname(const std::string_view input) {
-    if (has_opaque_path) { return; }
+  bool url::set_pathname(const std::string_view input) {
+    if (has_opaque_path) { return false; }
     path = "";
```

Comment:
I don't think we need this line, since on unsuccessful operation, it will override the current value.

Reply:
I am not sure what you mean.

Reply:
You can omit it.